.travis.yml

```diff
@@ -14,9 +14,13 @@ matrix:
   fast_finish: true
   include:
     - php: 7.1
+    - php: 7.2
+    - php: nightly
+  allow_failures:
+    - php: nightly
 
 before_install:
-  - phpenv config-rm xdebug.ini
+  - '[[ "$TRAVIS_PHP_VERSION" == "nightly" ]] || phpenv config-rm xdebug.ini'
```

Comment:
Why don't we remove Xdebug config for `nightly`?

Reply:
It is absent in `nightly` now

Reply:
nightly does not have XDebug on Travis, because XDebug itself is often not compatible with nightly yet